app/src/main/java/com/sdp/movemeet/view/navigation/Navigation.java

```diff
@@ -100,12 +100,12 @@ public static void goToHome(View view) {
      * Starts the chat activity
      * @param view the View in which to start the activity
      */
-    public static void goToChat(View view) {
+    /*public static void goToChat(View view) {
```

Comment:
Why do you comment the goToChat ?

Reply:
I commented it because this item was removed from the navigation because it was no longer used since the general chat was mainly the first version of the chat and was used for testing.